library_generation/README.md

```diff
@@ -10,6 +10,8 @@ configuration file.
 - Java runtime environment (8 or above)
 - Apache Maven (used in formatting source code)
 - Python (3.11.6 or above)
+- Docker 
+- Git
```

Comment:
Why do we need `git`?

Reply:
There are some [post processing scripts](https://github.com/googleapis/sdk-platform-java/blob/main/library_generation/postprocess_library.sh#L136-L143) perform git operations, I don't think we should  continue this way and should refactor it in the future, but currently we do.